src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ConfirmationDialog.vue

```diff
@@ -74,8 +75,8 @@ export default defineComponent({
       this.showDialog = false;
     },
     accept() {
-      this.$emit("accept");
       this.showDialog = false;
+      setTimeout(this.$emit, 500, "accept");
```

Comment:
Please just comment above why the timeout is required

Reply:
Implemented the changes.